reaper/CHANGELOG.md

```diff
@@ -9,11 +9,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add `Reaper.fuseOff()`. [#491](https://github.com/EmergeTools/emerge-android/pull/491)
 
+- Add logging for non-404 errors [#531](https://github.com/EmergeTools/emerge-android/pull/531)
```

Comment:
@rbro112 oops i think this should go in the next release since 1.0.1 was already released.